Apsitvarkom.DataAccess/PollutedLocationDTOFileRepository.cs

```diff
@@ -23,6 +27,7 @@ private PollutedLocationDTOFileRepository(Stream stream)
             AllowTrailingCommas = true,
             PropertyNameCaseInsensitive = true
         };
+        _mapper = new MapperConfiguration(cfg => cfg.AddProfile<PollutedLocationProfile>()).CreateMapper();
```

Comment:
didn't find a way to use DI here as static factory is necessary to support both ways of the repository. maybe I am missing something?